src/librustdoc/clean/types.rs

```diff
@@ -339,6 +339,7 @@ impl ExternalCrate {
 }
 
 /// Indicates where an external crate can be found.
+#[derive(Debug)]
```

Comment:
Why the added Derive? Is it used someplace now? (I don't see a change anywhere in the code that looks like it uses it)